static/js-src/shared.js

```diff
@@ -27,4 +27,10 @@ gtag('config', 'UA-179341418-1');
 // End Google Analytics
 
 
-window.csClient.addFormSubmitListner();
+// Make sure that our form handler for XSRF token updates gets
+// registered after all the Shoelace form listeners are registered
+// so that failed form validation cancels the submit event before
+// our listener tries to programatically submit the form.
+// Ten seconds should be enough time, and there is no rush since
+// XSRF tokens don't expire until much more time has passsed.
```

Comment:
Should probably add a TODO to fix this in a more reliable way that avoids the 10 second timeout.

Reply:
Added